lib/public/views/Runs/RunsPerSimulationPass/RunsPerSimulationPassOverviewModel.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { RemoteData } from '/js/src/index.js';
+import { buildUrl } from '../../../utilities/fetch/buildUrl.js';
+import { detectorsProvider } from '../../../services/detectors/detectorsProvider.js';
+import { RunsOverviewModel } from '../Overview/RunsOverviewModel.js';
+import { ObservableData } from '../../../utilities/ObservableData.js';
+import { getRemoteData } from '../../../utilities/fetch/getRemoteData.js';
+
+/**
+ * Runs Per Simulation Pass overview model
+ */
+export class RunsPerSimulationPassOverviewModel extends RunsOverviewModel {
+    /**
+     * Constructor
+     * @param {Model} model global model
+     */
+    constructor(model) {
+        super(model);
+        this._detectors$ = detectorsProvider.physical$;
+        this._detectors$.bubbleTo(this);
+        this._simulationPass = new ObservableData(RemoteData.notAsked());
```

Comment:
Please suffix with $ the observable data

Reply:
changed